src/core/public/chrome/ui/header/collapsible_nav.tsx

```diff
@@ -125,8 +128,9 @@ export function CollapsibleNav({
   const lockRef = useRef<HTMLButtonElement>(null);
   const groupedNavLinks = groupBy(navLinks, (link) => link?.category?.id);
   const { undefined: unknowns = [], ...allCategorizedLinks } = groupedNavLinks;
-  const categoryDictionary = getAllCategories(allCategorizedLinks);
-  const orderedCategories = getOrderedCategories(allCategorizedLinks, categoryDictionary);
+  const filterdLinks = filterLinks(allCategorizedLinks);
```

Comment:
Since the `filterLinks` will be an async function in the future, I don't think we can get the filtered links just by call this function.

Reply:
Yes. I think I should define a new state and use `useEffect` to call this async function in future. Currently this is a temporary status due to uncertainty.